packages/contracts/contracts/maci/MACI.sol

```diff
@@ -53,4 +58,13 @@ contract MACI is Ownable, BaseMACI {
     poll.init();
     poll.transferOwnership(msg.sender);
   }
+
+  /// @notice Set RegistryManager for MACI
+  function setRegistryManager(address registryManagerAddress) public onlyOwner {
```

Comment:
what is the logic behind allowing this to be changed multiple times?

Reply:
It doesn't matter how many time this method can be called. Registry manager doesn't have any data that can be changed. It's only an access control contract.
And it's not possible to use it as immutable property because maci is deployed before registry manager.

Reply:
why can't it be deployed before MACI? but yea makes sense what you said

Reply:
It's because deploy tasks from `maci-contracts` has their own order and it won't be a good practice if we allow users to change it. It's possible just to override the step but in this case there will be deployed registry manager and maci at the same step (and this is also not good if we want to use incremental deployment).

Reply:
ahh I see that makes sense. what if it was deployed within 08-maci?

Reply:
So, here is the case about incremental deploy. If registry manager is deployed with maci, it won't be possible to skip deploy step if one of them is already deployed and second one is not.

Reply:
got it! guess we can think more about this at a later time in case